### DISCUSSION TITLE
Pubspec Override

### DISCUSSION THREAD

**Discussion by VobSquash** [onepub-dev/onepub#545] [Q&A] [answered]
Hi, When im doing localhost development, I will be making use of the pubspec_override.yaml file. This so that I dont have to keep pushing new updates of my 1 package, which is used in a 2nd package. 

However when I publish, I dont want the pubspec override file to be used. Ultimately package 1 and 2 will both be uploaded to onePub, and used by my client application. Is there a way I can exclude the override file from the published artifact when using dart pub publish.

**Comment by bsutton** [comment, marked as answer]
The dart pub command doesn't publish the override file so the system should
work exactly as you want.
